packages/core/src/codewhispererChat/controllers/chat/controller.ts

```diff
@@ -536,19 +524,8 @@ export class ChatController {
                     mandatory: true,
                     title: 'Prompt name',
                     placeholder: 'Enter prompt name',
-                    description: 'Use this prompt in the chat by typing `@` followed by the prompt name.',
-                },
-                {
-                    id: 'shared-scope',
-                    type: 'select',
-                    title: 'Save globally for all projects?',
-                    mandatory: true,
-                    value: 'system',
-                    description: `If yes is selected, ${promptFileExtension} file will be saved in ~/.aws/prompts.`,
-                    options: [
-                        { value: 'project', label: 'No' },
-                        { value: 'system', label: 'Yes' },
-                    ],
+                    description:
+                        'Use this prompt in the chat by typing `@` followed by the prompt name. Prompt will be saved in ~/.aws/amazonq/prompts.',
```

Comment:
Use the common home directory to make this description apply for Windows. There is a getHomeDirectory function. In Windows it can be C:\\User\.aws